securedrop/source_templates/locales.html

```diff
@@ -2,8 +2,9 @@
 <div class="menu">
   <input id="menu-1-checkbox" class="menu__checkbox visually-hidden" type="checkbox" >
   <label for="menu-1-checkbox" class="menu__trigger">
-    <span class="menu__trigger-arrow">▼</span>
+    <i class="fa fa-globe menu__trigger-icon"></i>
```

Comment:
So the source interface is intended to render well even with the Tor Browser security slider set to its safest setting. With the Tor Browser set to safest, unfortunately this icon does not render:

![screen shot 2018-03-08 at 9 23 48 am](https://user-images.githubusercontent.com/7832803/37165904-948e2a76-22b2-11e8-9a68-76a4fdbf9d52.png)

However converting the icon to a PNG should do the trick - that's what we did with the other icons on the source interface - see #1567 for more.

Reply:
Got it. Will do that. That answers my question of why some icons weren't visible in tor previously. Guess I fiddled with the safe settings.

Reply:
Btw, just curious, why isn't this needed in journalist interface?

Reply:
@redshiftzero thanks for clearing that out. If we are reducing the amount of JS code in journalist interface then I think replacing icons with PNGs is also an important thing to do.

Reply:
Totally, since you're making that PNG for the source interface anyway, if you elect to also use it on the journalist interface, it'd be a nice bonus to this PR :tada: (but not required for merge)

Reply:
Sure. I will use the globe PNG for both the interfaces.

Reply:
@redshiftzero done. I have added the svg file as well just in case sometime in future we think of resizing the image or something, it will be easier from svg file. Let me know if I should remove the svg file from the PR though. 😃